src/components/status-panel/not-ready-info.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default function NotReadyInfo( { statuses } ) {
+	return (
+		<div className="status">
+			<p>
+				<strong>
+					<span role="img" aria-label="Wait!">
+						✋
+					</span>
+					Getting ready
+				</strong>
+			</p>
+			<p>Please wait a moment while TestPress gets everything ready.</p>
+			<p>
+				{ statuses.docker === 'ready' ? '👍' : '👉' } Starting Docker…
+				<br />
+				{ /* TODO: Make this work */ }
+				{ /* { statuses.node === 'ready' ? '👍' : '👉' } Installing node… */ }
+				{ /* <br /> */ }
+				{ statuses.grunt === 'ready' ? '👍' : '👉' } Compiling assets…
+				<br />
+				{ statuses.wordpress === 'ready' ? '👍' : '👉' } Installing WordPress…
```

Comment:
Need to make the progress screen nicer as per https://github.com/pento/testpress/issues/43.